modules/marketo_ma_contact/src/Form/MarketoMaContactConfiguration.php

```diff
@@ -77,8 +77,8 @@ public function buildForm(array $form, FormStateInterface $form_state, ContactFo
 
     $form['enabled'] = [
       '#type' => 'checkbox',
-      '#title' => $this->t('Enable tracking'),
-      '#description' => $this->t('Set to "Enable tracking" to turn on tracking for this contact form.'),
+      '#title' => $this->t('Capture Data'),
```

Comment:
I updated it to what the webform module used. Could have also gone with `Unless checked, captured form information will not be sent to Marketo.` No d.o. issue.